imagersite/imagersite/templates/registration/registration_complete.html

```diff
@@ -0,0 +1,4 @@
+{% extends 'imagersite/base.html' %}
```

Comment:
This page should say something telling the user to activate their account by checking the email.